tests/testthat/test_autotest.R

```diff
@@ -41,4 +41,23 @@ test_that("autotest recognizes learner that does not use .get_weights()", {
   result = run_experiment(task, learner)
   expect_false(result$ok)
   expect_string(result$error, pattern = "get_weights was not called")
-})
\ No newline at end of file
+})
+
+test_that("autotest configure_learner works", {
+  task = tsk("spam")
+  learner = lrn("classif.debug", error_train = 1)
+  # keep only feat_all_* tasks to save time
+  exclude = "^feat_single|^missings|^weights|^utf8|^sanity"
+  result = run_autotest(learner, 20, exclude)
+  expect_false(result$ok)
+
+  # example changes the learner's parameter during the autotest
+  cfg_lrn = function(learner, task) {
```

Comment:
if this is the `configure_learner` function, shouldn't the test check that `error_train = 0` in one case?